File: Segundo_Corte/Areas_Y_Perimetros/Codigos_Python/Square.py
class Square:
    def _init_(self, lado):
        print("Creating Square with side {l}".format(l=lado))
        self.lado = lado
    def Perimetro(self) :
        return 4*self.lado
    def area(self):
        return self.lado*self.lado
    
Table = Square(25)
Card = Square(8)
Closet = Square(35)
        
print(Table.Perimetro())
print(Card.Perimetro())
print(Closet.Perimetro())

print(Table.area())
print(Card.area())
print(Closet.area())

File: Segundo_Corte/Areas_Y_Perimetros/Codigos_Java/Square.java
package Square;

public class Square {
  
    public double lado;
    public Square(double lado){
        System.out.println("Creating Square with side " + lado);
        this.lado= lado;
    }
    public double Perimetro(){
    return 4*this.lado;
}
    public double Area(){
        return this.lado*this.lado;
    }
    public static void main(String[] args) {
        Square Table = new Square(25);
        Square Card = new Square(8);
        Square Closet = new Square(35);
        
        System.out.println(Table.Perimetro());
        System.out.println(Card.Perimetro());
        System.out.println(Closet.Perimetro());
        
        System.out.println(Table.Area());
        System.out.println(Card.Area());
        System.out.println(Closet.Area());
    }
        
  }